src/main/java/shouty/Shouty.java

```diff
@@ -15,13 +15,17 @@ public void shout(String person, String message) {
       messages.put(person, message);
     }
 
-    public List<String> getMessagesHeardBy(String listener) {
-        List<String> messagesHeard = new ArrayList<String>();
+    public Map<String, String> getMessagesHeardBy(String listener) {
```

Comment:
I agree, but we don't have any drive for ordering from the tests yet, so it doesn't matter. What this does is make an exercise later in the day (where you need to know who has shouted a message) possible.